src/library_gl.js

```diff
@@ -1400,6 +1411,97 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
```

Comment:
This addition duplicates all of the query-related code. Let's remove this and move the query functions outside `#if USE_WEBGL2=1` guard.

Then to get these working via the `EXT` suffixes, use routing functions like this:

    glGenQueriesEXT: 'glGenQueries',
    glDeleteQueriesEXT: 'glDeleteQueries',
    glIsQueryEXT: 'glIsQuery',
    glBeginQueryEXT: 'glBeginQuery',
    glEndQueryEXT: 'glEndQuery',
    glGetQueryivEXT: 'glGetQueryiv',
    glGetQueryObjectuivEXT: 'glGetQueryObjectuiv',

The `glGenQueriesEXT__sig` should not then be needed, since the compiler should know that `glGenQueriesEXT__sig == glGenQueries__sig` when it is doing the routing.